README.md

```diff
@@ -37,7 +37,7 @@ The `src` directory contains three files:
 The `tools` directory contains a util script used for generating a regex used in the utils
 module.
 
-The `test` directory contains unit tests, based on Karma and Jasmine, as well as HTML files
+The `test` directory contains unit and data-driven tests, based on Karma and Jasmine, as well as HTML files
```

Comment:
Should you add a small description about the differences between the tests? Just so we know what kind of test to put in which folder in the future?

Reply:
That makes a lot of sense. I'll expand the doc. Thanks for the suggestion!

Reply:
Would it be a problem to update the doc in a future pr? I have a separate [ticket](https://bugs.chromium.org/p/chromium/issues/detail?id=1303847) for it and I believe it would be easier to review after the test infra code is merged, so the reviewer has more context.

Reply:
Of course!